plugins/search-backend-module-elasticsearch/alpha-api-report.md

```diff
@@ -0,0 +1,61 @@
+## API Report File for "@backstage/plugin-search-backend-module-elasticsearch"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { BackendFeature } from '@backstage/backend-plugin-api';
+import { SearchQuery } from '@backstage/plugin-search-common';
+
+// @public
+export type ElasticSearchConcreteQuery = {
+  documentTypes?: string[];
+  elasticSearchQuery: Object;
+  pageSize: number;
+};
+
+// @public
+export type ElasticSearchCustomIndexTemplate = {
+  name: string;
+  body: ElasticSearchCustomIndexTemplateBody;
+};
+
+// @public
+export type ElasticSearchCustomIndexTemplateBody = {
+  index_patterns: string[];
+  composed_of?: string[];
+  template?: Record<string, any>;
+};
+
+// @public (undocumented)
+export type ElasticSearchHighlightConfig = {
+  fragmentDelimiter: string;
+  fragmentSize: number;
+  numFragments: number;
+  preTag: string;
+  postTag: string;
+};
+
+// @public
+export type ElasticSearchQueryTranslator = (
+  query: SearchQuery,
+  options?: ElasticSearchQueryTranslatorOptions,
+) => ElasticSearchConcreteQuery;
```

Comment:
These are supposed to go on the main export right?

Reply:
It should, but we were running into issues where some of the alpha exports used these types which meant we had to re-export from alpha export as well.. 🤔 do you have any idea of how to get around that?